### PR TITLE
Feature/graphvizaddition

### PR DESCRIPTION
**Pull request by zenln**

### CODE REVIEW

File: var/spack/repos/builtin/packages/graphviz/package.py [hunk 1]
```diff
@@ -29,11 +29,20 @@
...
+    url      = "http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz.tar.gz"
 
     # This commit hash is tag='stable_release_2.40.1'
     version('2.40.1', git='https://gitlab.com/graphviz/graphviz.git',
             commit='67cd2e5121379a38e0801cc05cce5033f8a2a609')
+    version('2.40.0', '4ea6fd64603536406166600bcc296fc8')
+    version('2.38.0', '5b6a829b2ac94efcd5fa3c223ed6d3ae')
+
+    
+    homepage = 'http://www.graphviz.org'
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/graphviz/package.py]: seems to be a duplicate of line 33

File: var/spack/repos/builtin/packages/graphviz/package.py [hunk 2]
```diff
@@ -29,11 +29,20 @@
...
+    homepage = "http://www.graphviz.org"
+    url      = "http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz.tar.gz"
 
     # This commit hash is tag='stable_release_2.40.1'
     version('2.40.1', git='https://gitlab.com/graphviz/graphviz.git',
             commit='67cd2e5121379a38e0801cc05cce5033f8a2a609')
+    version('2.40.0', '4ea6fd64603536406166600bcc296fc8')
+    version('2.38.0', '5b6a829b2ac94efcd5fa3c223ed6d3ae')
+
+
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/graphviz/package.py]: lots of whitespace in here

Comment (baberlevi) [inline var/spack/repos/builtin/packages/graphviz/package.py]: seems to be a lot of extra white space

**Comment by alalazo** [comment]
Closed and reopened to let CI run

**Comment by adamjstewart** [comment]
Ping @zenln

File: var/spack/repos/builtin/packages/graphviz/package.py [hunk 3]
```diff
@@ -29,11 +29,15 @@
 
 class Graphviz(AutotoolsPackage):
     """Graph Visualization Software"""
-    homepage = 'http://www.graphviz.org'
+
+    homepage = "http://www.graphviz.org"
+    url      = "http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz.tar.gz"
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/graphviz/package.py]: I fixed the `flake8` errors. Maybe I am missing the obvious, but how are we going to select among the two versions below, given that `url` seems not to have any version information?

**Comment by baberlevi** [comment]
I don't think we need the old versions for any particular reason right now, so closing.

**Pull request closed without merging**